x/airdrop/keeper/keeper_test.go

```diff
@@ -17,8 +17,6 @@ var (
 	feeCollectorAddress = authtypes.NewModuleAddress(authtypes.FeeCollectorName)
 	addr1               = sdk.AccAddress([]byte("addr1_______________"))
 	addr2               = sdk.AccAddress([]byte("addr2_______________"))
-	addr3               = sdk.AccAddress([]byte("addr3_______________"))
```

Comment:
Looks like something went wrong in the merge here I guess. I suspect you did not intend to do any changes to this module :wink:

Reply:
I clicked the wrong button to merge..then reversed the merge